mitmproxy/contentviews/msgpack.py

```diff
@@ -44,7 +83,7 @@ class ViewMsgPack(base.View):
     def __call__(self, data, **metadata):
         data = parse_msgpack(data)
         if data is not PARSE_ERROR:
-            return "MsgPack", format_msgpack(data)
+            return "MsgPack", (line for line in format_msgpack(data))
```

Comment:
Contentviews are expected to return an iterator over `TViewLine`. Just returning `format_msgpack(data)` as-is should be fine as lists are iterable. :)